main.py

```diff
@@ -1,16 +1,153 @@
+import asyncio
+import base64
+import datetime
+import glob
+import json
 import os
+import ssl
+from asyncio import Lock
+
+import certifi
 import decky_plugin
```

Comment:
This is a nitpick, but modern decky interface is provided with `import decky` instead of `import decky_loader`, thought decky-loader provides the exact same import for both really, so it shouldn't matter.

Reply:
Good to know, I changed it to `decky`.